src/ontology/mondo-edit.obo

```diff
@@ -420307,6 +420309,73 @@ intersection_of: MONDO:0001328 ! thyroid hormone resistance syndrome
 intersection_of: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/11799 ! THRB
 relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/11799 {source="Orphanet:566243"} ! THRB
 
+[Term]
+id: MONDO:0034846
+name: primary desmosis coli
+def: "A rare intestinal disease characterized by congenital partial or complete lack of the collagen mesh network in the intestinal wall, resulting in hypoperistalsis or aperistalsis. The enteric nervous system is normal or near-normal in the affected areas, although hypo- and dysganglionosis may be found in some proximal segments of the colon and/or small bowel. Patients present with chronic intractable slow transit constipation." [Orphanet:565641]
+subset: ordo_disease
+xref: ICD10:K59.9 {source="Orphanet:565641/ntbt"}
+xref: Orphanet:565641 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005020 {source="Orphanet:565641"} ! intestinal disease
+
+[Term]
+id: MONDO:0034849
+name: laminin subunit alpha 2-related limb-girdle muscular dystrophy R23
```

Comment:
what is this? is this a standard naming convention? is this just an unusual name for https://www.omim.org/entry/618138

Reply:
Oh, I didn't catch that. I'll add this as an xref on the OMIM term- thanks!